CONTRIBUTING.md

```diff
@@ -0,0 +1,292 @@
+# Contributing to c_sw
+
+NOTE: If you are reading this with a plain text editor, please note that this 
+document is formatted with Markdown syntax elements.  See 
+https://www.markdownguide.org/cheat-sheet/ for more information. It is
+recommended to view this document on [GitHub](https://github.com/NOAA-GSL/SENA-c_sw)
+
```

Comment:
The exact url might change once this PR is merged into develop, but we should reference this file directly: [GitHub](https://github.com/NOAA-GSL/SENA-c_sw/blob/develop/CONTRIBUTING.md)

Reply:
Good catch!  I copied and pasted from the README.md file, which is automatically displayed when you navigate to the github page.  But, CONTRIBUTING.md is not, so I do need to change this.

Reply:
Should be fixed now.